src/OrleansRuntime/Messaging/OutboundMessageQueue.cs

```diff
@@ -97,14 +95,13 @@ public bool SendMessage(Message msg)
             // First check to see if it's really destined for a proxied client, instead of a local grain.
             if (messageCenter.IsProxying && messageCenter.TryDeliverToProxy(msg))
             {
-                return true;
+                return;
             }
 
             if (!msg.ContainsHeader(Message.Header.TARGET_SILO))
             {
                 logger.Error(ErrorCode.Runtime_Error_100113, "Message does not have a target silo: " + msg + " -- Call stack is: " + (new System.Diagnostics.StackTrace()));
                 messageCenter.SendRejection(msg, Message.RejectionTypes.Unrecoverable, "Message to be sent does not have a target silo");
-                return true;
```

Comment:
GH in browser review is simply so awful! I just can't believe how bad it is!

This seems like a small easy PR so I almost hit the merge button, as all looked good.
Last moment decided to double check again, checked out your branch locally, pushed into code flow and immediately spotted the missing return. I sure did not see that missing return when reviewing in the browser.

This is awful!
Never going to do code reviews directly in GH. Its just not possible to not missing something.

How people are doing code reviews with GH? There **MUST** be some other tool that plugs in easily. Checking out and pushing in to code flow is too cumbersome.